### ISSUE TITLE
Missing "container.service_subscriber" tag after #48027

### ISSUE THREAD

**Issue by leofeyer** [symfony/symfony#48386]
The change in #48027 causes the following error for us:

```
Symfony\Component\DependencyInjection\Exception\InvalidArgumentException:
Service "Contao\CoreBundle\Controller\BackendController" misses a "container.service_subscriber"
tag with "key"/"id" attributes corresponding to entry "router" as returned by
"Contao\CoreBundle\Controller\BackendController::getSubscribedServices()".
```

Does this mean we now have to add tags for every service that is returned by `getSubscribedServices()`? I figured that is what autoconfiguration was for.

_Originally posted by @leofeyer in https://github.com/symfony/symfony/issues/48019#issuecomment-1330409829_

**Comment by stof** [comment]
Is your `BackendController` sevrice autowired or no ?

**Comment by leofeyer** [comment]
We are not using autowiring, because we are a bundle and not the application. We do use autoconfiguration, tough.

**Comment by leofeyer** [comment]
Closing this in favor of #48388 and #48404.
